extra/changelog-generator/changelog-generator

```diff
@@ -148,14 +148,15 @@ for repo in repos:
                 title = line
                 title_fetched = True
 
-            match = re.match("^ *Changelog: *(.*)", line, re.IGNORECASE)
-            if match:
+            match_changelog = re.match("^ *Changelog: *(.*)", line, re.IGNORECASE)
+            if match_changelog:
                 if log_entry:
                     add_entry(sha, log_entry)
                     log_entry = ""
                 log_entry_local = False
 
-                if re.match("^(Title|Commit|None)[ .]*$", match.group(1), re.IGNORECASE):
+                match_keyword = re.match("^(Title|Commit|All|None)[ .:]*(.*)$", match_changelog.group(1), re.IGNORECASE)
```

Comment:
Unless we really can't because of dependabot, I think we should require colon after the tag, otherwise this entry will be misinterpreted:

```
Changelog: All crashing bugs fixed
```

You might even consider a test case for this.

Reply:
Yes, I see what you mean, thanks.

I will replace it with: `^(Title|Commit|All|None)[ .:]+(.*)$`

dependabot uses something like:
````
Changelog:All : Bump bla bla bla
```

Reply:
@kacf It took me a while to fine tune this regex... I added more test cases that I hope cover the relevant corner cases.